Readability.js

```diff
@@ -497,11 +497,15 @@ Readability.prototype = {
               break;
           }
 
+          if (!this._isPhrasingContent(next)) break;
```

Comment:
Nit: please put the `break;` on the next line (indented, of course).

Reply:
I should probably tighten up the eslint rules to make this less frictionny, actually, so let's leave it for now and I'll try and sort it out at a later point.